app/lib/app/(public)/success_state/success_state.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:design_system/design_system.dart';
+import 'package:flutter/material.dart';
+import 'package:lottie/lottie.dart';
+
+class SuccessState extends StatelessWidget {
+  const SuccessState({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+    final size = MediaQuery.of(context).size;
```

Comment:
```suggestion
    final Size size = MediaQuery.sizeOf(context);
```
Vai evitar reconstruções desnecessárias pra cada mudança do `MediaQuery`